packages/shared/src/modules/contributors/libs/enums/contributor-order-by-key.enum.ts

```diff
@@ -0,0 +1,6 @@
+const ContributorOrderByKey = {
+	CREATED_AT: "created_at",
+	LAST_ACTIVITY_DATE: "last_activity_date",
```

Comment:
So as we agreed, maybe we should use camelCase here? (I think that this might be fixed in ticket regarding quality criteria, but if you'll fix anything else, you might to fix this also).

Reply:
@Vitaliistf can we fix this in your PR please? we need this pr to be merged to fix deployment

Reply:
Sure